content/04.study.md

```diff
@@ -473,3 +473,20 @@ However, because this model was not subsequently tested on ground-truth datasets
 
 Method development for interpreting new types of sequencing data has historically taken two steps: first, easily implemented hard cutoffs that prioritize specificity over sensitivity, then expert development of probabilistic models with hand-developed inputs [@tag:Torracinta2016_sim].
 We anticipate that these steps will be replaced by deep learning, which will infer features simply by its ability to optimize a complex model against data.
+
+
+
+### Neuroscience
+
+Artificial neural networks were originally conceived as a model for computation in the brain.
```

Comment:
In the intro we cite **A logical calculus of the ideas immanent in nervous activity** `[@doi:10.1007/BF02478259]`.  Can we reference that again in support of this sentence?